runtime/src/iree/base/internal/synchronization.h

```diff
@@ -57,9 +57,7 @@
 // https://github.com/llvm-mirror/compiler-rt/blob/master/include/sanitizer/tsan_interface.h
```

Comment:
you'll want to remove this NOTE comment too

Reply:
done